.github/workflows/full_ci.yml

```diff
@@ -187,6 +187,33 @@ jobs:
       - name: Execute run-docker.sh
         run: sh ./ci/run-docker.sh ${{ matrix.target }}
 
+  solaris:
```

Comment:
Could you add `solaris` to the `success` job (bottom of this file) so it becomes required for merge?